features/support/env.rb

```diff
@@ -3,3 +3,75 @@
 Before do
   @aruba_timeout_seconds = 10 # too slow on sloppy machines
 end
+
+After do
+  FileUtils.rm_rf('tmp')
+end
+
+def disable_linters
+  <<-content
+  AvoidOutlineForSingleExample:
```

Comment:
Ignore previous response. I'm not sure that there is a better way but I'm open to suggestions. The way the cukes currently work is that they required all linters but the linter-under-test to be disabled. I didn't want to muck with the the cukes because they use aruba, which I'm unfamiliar with and it was not immediately obvious how to do deal with. Since I was under a time-constraint, I opted to take the path of least resistance.

The most immediate way I can see to do this is create a new test_only method in gherkin_lint, disable_all, in the same way that there was an enable_all previously. I preferred this approach because the disable_all would require a little more precedence work and, against, we were working with a time-constraint.